File: templates/default.erl/PROJECTNAME.erl
-module(projectname).
-export([main/1]).
main(Args) ->
    io:format("Let's run Zproj.~n"),
    case Args of
        [FirstArg | _] -> 
            io:format("First argument: ~s~n", [FirstArg]);
        [] -> 
            ok
    end,
    init:stop(0).

File: templates/default.hs/PROJECTNAME.hs
module Main where
import qualified PROJECTNAME

import System.Environment (getArgs)
import System.Exit (exitSuccess)

main :: IO ()
main = do
    args <- getArgs
    case args of
        (firstArg:_) -> putStrLn ("First argument: " ++ firstArg)
        []           -> return ()
    exitSuccess